hack/community-gallery.sh

```diff
@@ -0,0 +1,24 @@
+#!/bin/bash
```

Comment:
For reference, here is internal Flatcar script which is used right now for building and publishing images to a community gallery: https://gist.github.com/invidian/41331399454b671f4f02ca693c731152. It also includes a staging gallery stage for testing.